### DISCUSSION TITLE
Cep telefonu bulan program

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus.github.io#68] [Soru Cevap]
Bir kişinin telefon numarası oldukça ilginç. Yedi haneli telefon numarasının son dört rakamını blok halinde ön tarafa taşıyınca yeni numara orjinal sayının iki katının bir fazlası oluyor. Bu kişinin telefon numarası nedir? Programı.

**Comment by DeniZZece** [comment]
```
#include<stdio.h>
int main()
{
	int a,b;
	for (a=100;a<1000;a++)
	{
		for (b=1000;b<10000;b++)
		{
			if ((a*10000+b)*2+1==(b*1000+a))
			{
				printf("%d%d",a,b);
			}
		}
	}
}
```
